ui-ngx/src/app/modules/home/pages/dashboard/dashboard-form.component.scss

```diff
@@ -14,5 +14,11 @@
  * limitations under the License.
  */
 :host {
+  .tb-form-panel.stroked {
+    gap: 0;
 
+    .mat-slide.fixed-title-width {
+      margin: 16px 0;
+    }
```

Comment:
This style can be replaced with a style from the platform's general styles in the file form.scss.